tests/common/test_pkg_config.py

```diff
@@ -9,20 +9,38 @@
 from pathlib import Path
 
 import httpx
+from jsonschema import Draft7Validator
 import pytest
 import pytest_check as check
 import yaml
 
 from msticpy.common import pkg_config
 from msticpy.context.geoip import GeoLiteLookup, IPStackLookup
 
-from ..unit_test_lib import custom_mp_config, get_test_data_path
+from ..unit_test_lib import custom_mp_config, get_test_data_path, get_queries_schema
 
 _TEST_DATA = get_test_data_path()
+_QUERIES_SCHEMA = get_queries_schema()
 
 # pylint: disable=protected-access
 
 
+def validate_queries_file_structure(query_file: Path, expected: bool = True):
```

Comment:
I think this really should be a test in the tests/data folder as its own test module..
The pkg_config tests are meant to test for people adding dependencies that are not set in setup.py or requirements (and  ensure that the requirements files all agree).

Reply:
Happy to add it as its own test module, but what test would like to see? I could add a few good & bad query files and check if they match the expected output?

Reply:
Maybe create something in tests/data/queries/test_query_files.py

Reply:
New test file created with sample query files!